### PR TITLE
[FIX]SUPA: fix bug during d2h copy and add missing function

### PR DESCRIPTION
**Pull request by Aaron20000101**
## Motivation and Context
fix bug in supa test code: fix bug during d2h copy and add missing function

## Description
fix bug during d2h copy and add missing function

## Checklist
**Before PR**:

- [ ] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: impl/supa/test/conform_test.cpp [hunk 1]
```diff
@@ -83,26 +83,30 @@ int32_t initLibrary() { return diopiSuccess; }
 
 int32_t finalizeLibrary() { return diopiSuccess; }
 
-int32_t buildGeneratorState(diopiContextHandle_t ctx, diopiTensorHandle_t out) { return diopiNoImplement; }
+#include "litert.hpp"
+
+int32_t buildGeneratorState(diopiContextHandle_t ctx, diopiTensorHandle_t out) {
```
Comment (yangbofun) [inline impl/supa/test/conform_test.cpp:88]: 这个返回类型写错了，应该是diopiError_t.

**Comment by wugeshui** [comment]
更新diopi最新的main到当前分支吧

**Comment by Aaron20000101** [comment]
> 更新diopi最新的main到当前分支吧

已更新

File: impl/supa/test/conform_test.cpp (hunk 1 above)
Comment (yangbofun) [reply impl/supa/test/conform_test.cpp:88]: 嗯嗯， 我后续提个pr专门改一下。

**Review by yangbofun — APPROVED** [review]

**Review by wugeshui — APPROVED** [review]

**Pull request merged**